classes/book.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :title, :author, :publisher, :label, :cover_state
+
+  def initialize(title, author, publisher, publish_date)
+    super(publish_date, false)
+    @title = title
+    @author = author
+    @publisher = publisher
+    @label = nil
+    @cover_state = 'bad'
+  end
```

Comment:
- [ ] author should not be a parameter
- [ ] @cover_state should not have a default of 'bad'